lib/matplotlib/tests/test_backend_ps.py

```diff
@@ -192,6 +193,17 @@ def test_type3_font():
     plt.figtext(.5, .5, "I/J")
 
 
+@image_comparison(["coloredhatcheszerolw.eps"])
+def test_colored_hatch_zero_linewidth():
+    ax = plt.gca()
+    ax.add_patch(Ellipse((0, 0), 1, 1, hatch='/', facecolor='none',
+                         edgecolor='r', linewidth=0))
+    ax.add_patch(Ellipse((0.5, 0.5), 0.5, 0.5, hatch='+', facecolor='none',
+                         edgecolor='g', linewidth=0.2))
+    ax.add_patch(Ellipse((1, 1), 0.3, 0.8, hatch='\\', facecolor='none',
+                         edgecolor='b', linewidth=0))
```

Comment:
Not a huge deal, but you could save quite a bit on the size of the eps file with `ax.set_axis_off()` (which allows skipping the encoding of all the text glyphs).

Reply:
Indeed! Went down from 11k to 4k.